woodwork/tests/utils/test_accessor_utils.py

```diff
@@ -37,6 +37,21 @@ def test_init_series_valid_conversion_specified_ltype(sample_series):
     assert series.ww.semantic_tags == set()
 
 
+def test_init_series_with_invalid_type(sample_df):
+    error_message = f'Input must be of series type. The current input is of type {type(sample_df)}'
+    with pytest.raises(TypeError, match=error_message):
+        init_series(sample_df)
+    error_message = f'Input must be of series type. The current input is of type {type(1)}'
+    with pytest.raises(TypeError, match=error_message):
+        init_series(1)
+    error_message = f'Input must be of series type. The current input is of type {type("string")}'
+    with pytest.raises(TypeError, match=error_message):
+        init_series("string")
+    error_message = f'Input must be of series type. The current input is of type {type(None)}'
+    with pytest.raises(TypeError, match=error_message):
+        init_series(None)
```

Comment:
We have a lot of duplicate code here. We could clean this up by putting all the input values we want to test inside a list and then looping over that list, updating the error message and calling init_series to make sure it works properly:
```
inputs = [sample_df, 1, "string", None]
for input in inputs:
    # Update error message with input
    # Call init_series inside with context mgr to check error is raised
```

Reply:
I understand, I have put it all within a loop